src/tr/yaoibar/src/eu/kanade/tachiyomi/extension/tr/yaoibar/Yaoibar.kt

```diff
@@ -0,0 +1,12 @@
+package eu.kanade.tachiyomi.extension.tr.yaoibar
+
+import eu.kanade.tachiyomi.multisrc.madara.Madara
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class Yaoibar : Madara(
+    "Yaoibar",
+    "https://yaoibar.gay/",
+    "tr",
+    dateFormat = SimpleDateFormat("MMMM dd, yyyy", Locale("tr")),
+)
```

Comment:
Did you check if it works with useNewChapterEndpoint set to true? Would be worth a shot. Including it would skip 1 request.